assignments/this.js

```diff
@@ -1,26 +1,69 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Window binding is when .this is referencing a value in the global environment. 
+* 2. Implicit binding is when whatever is to the left 
+* 3. New binding is when .this is used to create a new function based on a previous function.
+* 4. Explicit binding is when .this is used with .call .apply or .bind on a function.
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 
 // code example for Window Binding
+function win() {
+    return this;
+}
+
+//or
+console.log(this);
+console.log(window === this);
```

Comment:
First, `window` is only defined in the browser. In the console, it will be `global`
Second, the variable/unknown should be on the left, and the constant on the right, so `console.log(this === window)`